how-to-pitch/readme.md

```diff
@@ -0,0 +1,49 @@
+> Goal of this guide: Helping you to pitch
+
+# How to pitch
+
+"Trust me I got this" is one of the least reassuring sentences in the world. So how do you convey that you can do something without clearly saying that you "got diss"?
+
+<img src="https://media.giphy.com/media/XfxbrDXAkIqFa/giphy.gif" width="100%">
+
+At New Atoms we do quite some pitching. We pitch ideas for articles to clients and we pitch our service to everyone who will listen. But what makes a good pitch?
+
+## What the interwebs says
+
+The internet is a bit divided on what makes a good pitch. From all sources and some of our own experience we have distilled some of the most important things.
```

Comment:
perhaps mention something small about pitch decks here? these structures are amazingly clarifying to me